deployment_scripts/linode-marketplace-openbao/openbao-deploy.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+set -e
+DEBUG="NO"
+if [ "${DEBUG}" == "NO" ]; then
+  trap "cleanup $? $LINENO" EXIT
+fi
+
+##Linode/SSH security settings
+#<UDF name="user_name" label="The limited sudo user to be created for the Linode: *No Capital Letters or Special Characters*">
+#<UDF name="disable_root" label="Disable root access over SSH?" oneOf="Yes,No" default="No">
+# <UDF name="client_ips" label="List of IP addresses to whitelist" example="192.168.1.2, 192.168.1.3" default="" />
+
+## Domain Settings
+#<UDF name="token_password" label="Your Linode API token. Required for Private IP check/add and DNS records [if applicable]">
+#<UDF name="subdomain" label="Subdomain" example="The subdomain for the DNS record: www (Requires Domain)" default="">
+#<UDF name="domain" label="Domain" example="The domain for the DNS record: example.com (Requires API token)" default="">
+#<UDF name="soa_email_address" label="email for SOA" default="">
```

Comment:
```suggestion
```
we don't use this variable anywhere in the playbook

Reply:
this is for if the customer wants to create dns records